timed-content-module/includes/frontend.css.php

```diff
@@ -18,34 +18,34 @@
 <?php
 } else {
 	if ( ! $display && 'msg' != $settings->expire_content_action ) { ?>
-		.fl-module-timed-content-module.fl-node-<?php echo $id; ?> {
+		.fl-module-timed-content-module.fl-node-<?php echo esc_attr($id); ?> {
 			display : none;
 		}
 <?php }
 }
 
 if ( ! empty( $settings->timed_msg_font ) && 'Default' != $settings->timed_msg_font['family'] ) { ?>
-.fl-node-<?php echo $id; ?> .timed-content-message p {
+.fl-node-<?php echo esc_attr($id); ?> .timed-content-message p {
 	<?php FLBuilderFonts::font_css( $settings->timed_msg_font ); ?>
 }
 <?php } ?>
 
 .fl-node-<?php echo $id; ?> .timed-content-message p {
-	color: #<?php echo $settings->timed_msg_color; ?>;
```

Comment:
why is this removed `$settings->timed_msg_color;`

Reply:
I'll check and change.